### ISSUE TITLE
contributors.py parses only open issues and pull requests

### ISSUE THREAD

**Issue by bitnik** [jupyterhub/zero-to-jupyterhub-k8s#878]
Hi, `contributors.py` parses only open issues and pull requests now, because `tools.contributors.get_all_contributors` doesn't pass value for state and in github api state is `open` by default:
- https://developer.github.com/v3/pulls/#parameters
- https://developer.github.com/v3/issues/#parameters

I think it should parse all issues and closed pull requests:
```python
    issues = repo.get_issues(state='all')
    pulls = repo.get_pulls(state='closed')
```

Another problem is that `get_comments` is used to list comments of a pull request but it also returns review comments: https://github.com/PyGithub/PyGithub/blob/master/github/PullRequest.py#L535. As stated in the warning `get_issue_comments` must be used to get comments.

What do you think? If you aggree, I can make a PR to do updates.

**Comment by consideRatio** [comment]
@bitnik wooo nice summary and catch! A PR would be awesome!!! :100: 

![image](https://user-images.githubusercontent.com/3837114/44737950-afeb2c00-aaf3-11e8-9762-0aeac658d86c.png)

**Comment by minrk** [comment]
Closed by #881
